### DISCUSSION TITLE
Next.js links stop working after a navigating a several links

### DISCUSSION THREAD

**Discussion by acomanescu** [vercel/community#4030] [Help]
### Summary

I created a dashboard application with many side navigation items. While checking that everything works correctly, I've noticed that when deployed on Vercel (not locally) the links stop working after navigation through multiple links. I can see the network request after clicking the link, but nothing happens. I'm using the latest canary 13.4.20-canary.24 with app router only. When running locally with `npm run dev` or `npm run build && npm run start` it works as expected. The issue happens only when deployed on Vercel.

Thank you!

### Steps to Reproduce

I cannot share a reproducible example, but the issue happens after navigation through 15-20 links, then it stops working.

**Comment by acomanescu** [comment]
Downgrading to `13.4.19` seems to solve this issue. I have to mention that the navigation issue happens when changing the links with a ~2second delay in between.

**Comment by amyegan** [comment]
Hey @arobert93. I wasn't able to replicate this problem in my test app. If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will help with debugging. 🙏

**Comment by ahmadworks** [comment]
Same problem here after updating to v13.5.2 stable.
Only happens after deploying to Vercel, switching between links multiple times will make them stop working and you can't use links until reloading the page, now logs, errors or any network attempts when clicking links.

Downgrading to 13.4.19 solved the problem.

**Comment by mattkauffman23** [comment]
Seeing this as well. We use app router and make some use of parallel and intercepting routes. Not sure if that's related. Anyone know if there is an issue tracking this problem?

**Comment by amyegan** [comment]
I have not been able to reproduce this problem in my test app. If anyone can share [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will let me see what's happening. 🙏

**Comment by ahmadworks** [comment]
I submitted a ticket to Vercel with 2 URLs where you can reproduce the issue with 13.5.2 and compare it with previous versions.

> I have not been able to reproduce this problem in my test app. If anyone can share [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will let me see what's happening. 🙏
